src/product/README.md

```diff
@@ -0,0 +1,28 @@
+# Product library
+
+Product library merges multiple contracts to a single product contract.
+
+## Configuration
+
+Sometimes we can get names collisions when merging multiple contracts. In that case it's recommended to use the configuration file in the following format:
+
+```json
+{
+  "replacements": [
+    {
+      "filename": "tests/product/static/remote_collisions11.scilla",
+      "line": 19,
+      "replacee": "m_f",
+      "replacement": "remoteCollisions12_m_f"
```

Comment:
Would syntax like this work? It feels it would be a bit easier to write:

```suggestion
      "filename": "tests/product/static/remote_collisions11.scilla",
      "line": 19,
      "m_f": "remoteCollisions12_m_f"
```

Reply:
Could it be that two replacements on the same line are needed ?

Reply:
> Would syntax like this work? 

Yes, we can use that syntax, but it will complicate processing of this config by tools (in our code or if the user decide to somehow automate generation of the JSON config).

This will also make it not possible to use `ppx_deriving_yojson` to generate a config.

Maybe just left it as it is?

Reply:
> Could it be that two replacements on the same line are needed ?

Done

Reply:
Ok, let's leave it as it is then.